src/views/News/components/NewsList/index.jsx

```diff
@@ -77,6 +39,46 @@ const NewsList = ({
     };
   });
 
+  //https://www.pluralsight.com/guides/re-render-react-component-on-window-resize
+  //Excellent resource for handling rerender on resize -Josh
+
+  const singleHeader = (
+    <div className={styles.header}>
+      <Grid container direction="row">
+        <Grid item xs={12}>
+          <Typography variant="body2" className={styles.header}>
+            {header}
+          </Typography>
+        </Grid>
+      </Grid>
+    </div>
+  );
+
+  const fullHeader = (
+    <Grid container direction="row" className={styles.header}>
+      <Grid item xs={2}>
+        <Typography variant="body2" className={styles.header}>
+          CATEGORY
+        </Typography>
+      </Grid>
+      <Grid item xs={5}>
+        <Typography variant="body2" className={styles.header}>
+          SUBJECT
+        </Typography>
+      </Grid>
+      <Grid item xs={3}>
+        <Typography variant="body2" className={styles.header}>
+          POSTED BY
+        </Typography>
+      </Grid>
+      <Grid item xs={2}>
+        <Typography variant="body2" className={styles.header}>
+          POSTED
+        </Typography>
+      </Grid>
+    </Grid>
+  );
```

Comment:
Why did you move this inside of the component? If it is constant and not dependent on params/members of the component, it should be outside where it was.